### DISCUSSION TITLE
tmux pane borders are not visible in WSL with Ubuntu

### DISCUSSION THREAD

**Discussion by JanKowalik** [microsoft/WSL#9500] [General]
### Version

Microsoft Windows [Version 10.0.22621.963]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

Linux version 5.15.79.1-microsoft-standard-WSL2 (oe-user@oe-host) (x86_64-msft-linux-gcc (GCC) 9.3.0, GNU ld (GNU Binutils) 2.34.0.20200220) #1 SMP Wed Nov 23 01:01:46 UTC 2022

### Distro Version

Ubuntu 20.04

### Other Software

tmux 3.0a

### Repro Steps

1. Open Ubuntu on WSL.
2. Open tmux
3. Split the window into panes vertically or horizontally

### Expected Behavior

Tmux pane borders are visible

### Actual Behavior

Tmux pane borders are not displayed

![image](https://user-images.githubusercontent.com/36519208/210519506-168a5903-8719-4892-b548-c040171281e5.png)

**Comment by elsaco** [comment]
@JanKowalik does it do the same with a different color scheme?

This screenshot is with the default color scheme:
![tmux_ubuntu2004](https://user-images.githubusercontent.com/3933920/210649559-f9bc8e86-2f11-4956-91dc-e2eaed54e316.PNG)

**Comment by JanKowalik** [comment]
hmm.. Not sure if that was the default. But after changing to the first theme on the list - Campbell, it shows the borders now. Thanks.

**Comment by JanKowalik** [comment]
oh.. that was not true. Actually I notice now that it shows the borders using either theme when I view it on my laptop screen. It does not display it on my attached monitor though. How odd. Do you have any idea why that could be?
